pyproject.toml

```diff
@@ -1,11 +1,12 @@
 [project]
 name = "pelican-minify"
-version = "2.0.0"
+version = "2.0.1"
```

Comment:
Because [AutoPub](https://justinmayer.com/projects/autopub/) automatically increments version numbers, there is no need to do so manually. And in fact, doing so manually would result in skipped version numbers, so please revert this manual change to the version number.